modules/xmpp/ChatRoom.js

```diff
@@ -689,6 +689,31 @@ export default class ChatRoom extends Listenable {
 
     /**
      *
+     * @param id
+     * @param body
+     * @param nickname
+     */
+    sendPrivateMessage(id,body, nickname) {
+        const msg = $msg({ to: this.roomjid+"/"+id,
+            type: 'chat' });
+
+        msg.c('body', body).up();
+        if (nickname) {
+            msg.c('nick', { xmlns: 'http://jabber.org/protocol/nick' })
+                .t(nickname)
+                .up()
+                .up();
+        }
+        msg.c('x', { xmlns: 'http://jabber.org/protocol/muc#user' })
```

Comment:
Look like this line isn't needed, for an unknown reason, this element is already send when using _send_ method below.

Reply:
If it's not needed... drop it then?